db/migrate/20241020010245_add_tool_name_to_ai_tools.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddToolNameToAiTools < ActiveRecord::Migration[7.1]
+  def up
+    add_column :ai_tools, :tool_name, :string, null: false, default: "", if_not_exists: true
```

Comment:
last one and I will merge it, this need a max length, sorry it is taking me so long here.